ran-core/src/main/java/io/ran/MappingClassWriter.java

```diff
@@ -17,20 +17,21 @@
 
 public class MappingClassWriter extends AutoMapperClassWriter {
 	Clazz mapperClazz;
-	public MappingClassWriter(Class clazz) {
+	public MappingClassWriter(Class clazz) throws NoSuchMethodException {
```

Comment:
Why will the constructor throw this exception? Should it not handle it internally (as I guess it should not happen - but the java compiler doesn't know this).